libcxx/test/std/algorithms/alg.modifying.operations/alg.swap/iter_swap.pass.cpp

```diff
@@ -17,28 +17,37 @@
 #include <cassert>
```

Comment:
Above in the comment:

```
// template<Iterator Iter1, Iterator Iter2>
//   requires HasSwap<Iter1::reference, Iter2::reference>
// void iter_swap(Iter1 a, Iter2 b); // constexpr since C++20
```